python/test/segyio_c.py

```diff
@@ -182,7 +182,7 @@ def test_custom_iline_xline():
         segyfile(str(testdata / 'small.sgy'), "r", 0).segyopen(xline=241)
 
     f = segyfile(str(testdata / 'small.sgy'), "r", 0).segyopen(iline=9, xline = 17)
-    layout = f.traceheader_layout()
+    layout = f.traceheader_layouts()
 
     standard_header_layout = layout["SEG00000"]
```

Comment:
If we keep `layouts`, then maybe also update the temporary variable:
```suggestion
    layouts = f.traceheader_layouts()

    standard_header_layout = layouts["SEG00000"]

Reply:
Done, but as I rebased, it is not a fixup :)